src/utils/connection.ts

```diff
@@ -10,7 +10,9 @@ import { IStorage } from "../types";
 export function handleWsDashboard(storage: IStorage) {
   return (data: any) => {
     // update network height
-    storage.setCurrentHeight(data.best_block_height as number);
+    const height = data.best_block_height as number;
+    storage.setCurrentHeight(height);
+    storage.unlockUtxos(height);
```

Comment:
This ws message is sent every second. I think we should check if the height has changed before doing anything here, what do you think?

Reply:
Started to check if the current height is different than the new height before calling `unlockUtxos`